lib/data/network/prayer_times_remote_source.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:math';
+
+import 'package:clock/clock.dart';
+import 'package:injectable/injectable.dart';
+import 'package:mcnd_mobile/data/models/api/prayer_time_filter.dart';
+import 'package:mcnd_mobile/data/models/app/day_prayer.dart';
+import 'package:mcnd_mobile/data/models/mappers/mapper.dart';
+import 'package:mcnd_mobile/data/network/mcnd_api.dart';
+
+@lazySingleton
+class PrayerTimesRemoteSource {
+  final Clock _clock;
+  final McndApi _api;
+  final Mapper _mapper;
+
+  PrayerTimesRemoteSource(this._api, this._mapper, this._clock);
+
+  Future<List<DayPrayers>> fetchThisMonthPrayers() async {
```

Comment:
return type here should be just list!

Reply:
we are talking to `_api.getPrayerTime` which is async